dll/win32/syssetup/wizard.c

```diff
@@ -2046,6 +2147,11 @@ RegistrationProc(LPVOID Parameter)
     RegistrationData->Registered = 0;
     RegistrationData->DefaultContext = SetupInitDefaultQueueCallback(RegistrationData->hwndDlg);
 
+#ifdef __REACTOS__ /* HACK */
```

Comment:
Why is this done only now, and not when other data font is set up (currently done in usetup)?

Reply:
OK.